tests/test_core.py

```diff
@@ -1,17 +1,14 @@
 """Tests standard tap features using the built-in SDK tests library."""
 import re
+from os import path
 from unittest import mock
 
 import pytest
-from os import path
 from singer_sdk.testing import get_standard_tap_tests
 
 from tap_smoke_test.tap import TapSmokeTest
 
-FIXTURE_DIR = path.join(
-    path.dirname(path.realpath(__file__)),
-    "../../demo-data",
-)
+FIXTURE_DIR = "./demo-data"
```

Comment:
Does Pytest set the current working directory? If not, then we should still use a path relative to `__file__` to ensure it can be run from directories other than the repository root.